PetCenter/PetCenter/Domain/Model/Review.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PetCenter.Domain.Model
+{
+    public class Review(Offer offer, int grade, string comment)
+    {
+        public Offer Offer { get; set; } = offer;
```

Comment:
The review doesn't need to see the offer.

Reply:
Fixed.